generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -762,7 +762,12 @@ public class UserService {
             // if IdP sends last updated information, use it to determine if an update should happen
             if (details.get("updated_at") != null) {
                 Instant dbModifiedDate = existingUser.get().getLastModifiedDate();
-                Instant idpModifiedDate = new Date(Long.valueOf((Integer) details.get("updated_at"))).toInstant();
+                Instant idpModifiedDate;
```

Comment:
nut: can we move this to a separate function?

Reply:
@sendilkumarn I just simplified the logic since this seems to only affect Okta. Please have a look.